docs/cloudformation/federation.template.yaml

```diff
@@ -0,0 +1,60 @@
+AWSTemplateFormatVersion: 2010-09-09
+Description: Janus integration for login federation
+
+# !! This template is provided as an example and is not intended for production use !!
+--!! This line invalidates the template and will need to be removed before CloudFormation !!--
+
+Parameters:
+  JanusInstanceRoleArn:
+    Description: ARN of Janus' IAM role
+    Type: String
+    AllowedPattern: "arn:aws:iam::[0-9]{12}:.+"
+    ConstraintDescription: Should be an AWS IAM role
+  JanusDevUserArn:
```

Comment:
While I understand the need for this, it does punch a hole in the security of the application. I think I'd prefer to separately document how to do this for people that want to, rather than have it be explicitly a part of the template. I think this makes it too easy for people to give Janus access to an IAM user, which is somewhat at odds with the spirit of the application. In particular, it's unlikely that this is needed "be default". Partly because not everyone will use it, but also because even if they do, they are unlikely to use it for all stages or even all AWS accounts. As such, making this an exception rather than the norm might help guide users towards making better use of Janus.

Very happy to chat this one over with you. The local development issue is definitely something important to consider and I do agree that this approach is a good solution to that problem.

Reply:
Very good points! ⭐  
I think it is best to remove the idea of a Dev user and wait to attempt it in the future if this is something that we want to support. I agree that it would actually be better as the exception, as end-to-end testing should be possible with access to only a couple of accounts.